docs/graphcache/gotchas.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Gotchas
+order: 5
+---
+
+# Gotchas
+
+When using this normalised cache there are a few out of the box features
+we need to be aware of, being aware of these features will make it easier
+to define best practices and to avoid pitfalls.
+
+Most of the measures we'll cover are brought to life to ensure we can build out
+to a fully offline normalised cache.
+
+## Commutativity
+
+By default the cache ensures idempotent and commutative results, the last part of this
+is the most relevant one to be aware of.
+
+Commutative guarantees mean that we'll keep the order of operations consistent, let's look
```

Comment:
I think the use of “we” to mean “Graphcache” is a little confusing in this particular document.

Since the docs use “we” for instructions, we’ll likely have to resort to explicitly saying that “Graphcache” does something rather than using “we” for “us / the library” too